main.cpp

```diff
@@ -20,13 +20,17 @@
 #include <string.h>
 #include "KVStore.h"
 #include "kvstore_global_api.h"
+#include "DeviceKey.h"
 
 using namespace mbed;
 
 #define EXAMPLE_KV_VALUE_LENGTH 64
 #define EXAMPLE_KV_KEY_LENGTH 32
 #define err_code(res) MBED_GET_ERROR_CODE(res)
 
+#define STR_EXPAND(tok) #tok
+#define STR(tok) STR_EXPAND(tok)
```

Comment:
This is copied from https://github.com/ARMmbed/mbed-os/blob/f2278567d09b9ae9f4843e1d9d393526b9462783/storage/kvstore/kv_config/source/kv_config.cpp#L170-L171
It doesn't look pretty but we need it, see below.